06_deploy_masters.sh

```diff
@@ -7,12 +7,18 @@ set -eux
 
 export OS_TOKEN=fake-token
 export OS_URL=http://ostest-api.test.metalkube.org:6385/
+
+# Clean previously env 
+for node in `openstack baremetal node list | grep master | awk '{print $2}'`; do  openstack baremetal node undeploy $node; done 
+for node in `openstack baremetal node list | grep master | awk '{print $2}'`; do  openstack baremetal node delete $node; done 
+
```

Comment:
It'd be quicker to do this in a single loop with jq (see line 18 below) to avoid doing the baremetal node list twice?